src/main/java/com/google/step/snippet/external/StackOverflowClient.java

```diff
@@ -110,35 +105,12 @@ private String getTitle(String questionId) {
   /* Get the content of the answer and store it in the card. */
   private String getAnswerBody(String answerId) {
     String answerUrl = String.format(ANSWER_URL_TEMPLATE, answerId);
-    return getResponse(answerUrl, BODY_PARAMETER);
-  }
-
-  /* Return the description parsed from answer body. */
-  private String getDescription(String body) {
-    Document doc = Jsoup.parse(body);
-    // Combine all description in the answer body.
-    Elements descriptionHtml = doc.select("p");
-    String description = "";
-    for (Element e : descriptionHtml) {
-      description += e.outerHtml();
-      if (description.length() >= MAX_DESCRIPTION_LENGTH) {
-        description = description.substring(0, MAX_DESCRIPTION_LENGTH);
-        break;
-      }
-    }
-    return description;
-  }
-
-  /* Return the code parsed from answer body. */
-  private String getCode(String body) {
-    Document doc = Jsoup.parse(body);
-    // Combine all code in the answer body.
-    Elements codeHtml = doc.select(CODE_PARAMETER);
-    String code = "";
-    for (Element e : codeHtml) {
-      code += e.outerHtml();
+    String answerBody = getResponse(answerUrl, BODY_PARAMETER);
+    if (answerBody.length() > 300) {
```

Comment:
getResponse can return null. Add a null check here before trying to access length()